specification/search/data-plane/Azure.Search/preview/2019-05-06-preview/searchservice.json

```diff
@@ -7802,6 +7778,33 @@
       },
       "description": "The language codes supported for input by OcrSkill."
     },
+    "OcrSkillLineEnding": {
+      "type": "string",
+      "enum": [
+        "Space",
+        "CarriageReturn",
+        "LineFeed"
+      ],
+      "x-ms-enum": {
+        "name": "LineEndingOption",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "Space",
+            "description": "Space, which is the default value for line endings"
+          },
+          {
+            "value": "CarriageReturn",
+            "description": "Carriage return, represented by \r\n"
```

Comment:
This is actually CarriageReturnLineFeed. Windows uses CRLF, while Mac classes (many files may have been saved on it, or saved in compat mode) uses CR, while linux/new Mac uses LF. Shouldn't all those values be supported?

At the very least, I'd rename this (and its support) for what it is: `CarriageReturnLineFeed`. As named, it's misleading.

Reply:
@heaths these are some great points and suggestions.  I've done a quick investigation, and the person who wrote this code is no longer on the team.  But, since these values have existed as enums on our backend for some time with those names, I'm hesitant to change them without doing more research on their usage.  (Also, I would want to change the production codebase before changing this in the SDK.)  So I'll create a work item to investigate this further: https://msdata.visualstudio.com/Azure%20Search/_workitems/edit/760844.  But for this checkin, I'd like to keep the current values of the enum that are in the codebase.